.github/workflows/push.yaml

```diff
@@ -87,11 +91,40 @@ jobs:
         git add application/Chart.yaml
         git commit -m "[skip-ci] Update artifacts" 
 
+    # Push Chart.yml with Updated Version
     - name: Push changes
-      uses: ad-m/github-push-action@master
-      with:
-        github_token: ${{ secrets.STAKATER_GITHUB_TOKEN }}
-
+      shell: bash {0}
+      run: |
+        git config --local user.email "stakater@gmail.com"
```

Comment:
This should come from secret or variable. is it hard coded in other place?

Reply:
yes its hard coded in application chart,